app/volunteer-add-confirm/[[...volunteer-add-confirm]]/page.tsx

```diff
@@ -1 +1,39 @@
-// Write your code here
\ No newline at end of file
+// imports
+import { ButtonExit, ButtonSubmit, ButtonEdit } from "../../components/SurveyButtons";
+import UpdateInventoryBanner from "../../components/UpdateInventoryBanner";
+
+const AddConfirmPage: React.FC = () => {
+    
+    return (
+        <div>
+            {/* Imported Banner */}
+            <div>
+                <UpdateInventoryBanner/>
+            </div>
+            <div>
+                {/* Top Buttons */}
+                <div className="flex p-20 crimson-regular text-2xl content-center justify-between">
+                    <ButtonEdit/>
+                    <ButtonExit/>
+                </div>
+
+                {/* Central text */}
+                <div className="text-black crimson-bold flex text-4xl content-center justify-center text-center">
+                    This action will:
+                </div>
+
+                {/* Placeholder text */}
+                <div className="text-gray crimson-regular pt-10 flex text-4xl content-center justify-center text-center">
+                    Add [quantity] [units] of [itemName].
+                </div>
+                
+                {/* Bottom Buttons */}
+                <div className="flex pt-[250px] crimson-regular text-2xl justify-center">
```

Comment:
<img width="1440" alt="Screenshot 2024-11-15 at 7 08 44 PM" src="https://github.com/user-attachments/assets/303e7167-c381-4c1c-b243-3f079fbc0811">

[nitpick] Do you mind moving the submit button higher? 

Volunteer pages are expected to be on a desktop computer (stationed in the pantry for volunteers to use) so this is my bad because I don't think I clarified that with you! For iPad view this looks well spaced but since this is for standard computer view, if the submit button could be a little higher that'd be so phenomenal